test/server_integration/http/ssl_with_p12_intermediate/config.js

```diff
@@ -16,6 +17,7 @@ export default async function ({ readConfigFile }) {
   const certificateAuthorities = [readFileSync(CA1_CERT_PATH), readFileSync(CA2_CERT_PATH)];
 
   return {
+    testConfigCategory: ScoutTestRunConfigCategory.UI_TEST,
```

Comment:
same here, probably  `API_TEST`

Reply:
There's a follow-up PR for it already: #2107